src/in-app-browser/in-app-browser.tsx

```diff
@@ -0,0 +1,40 @@
+import {StatusBar} from 'react-native';
+import InAppBrowser, {
+  InAppBrowserOptions,
+} from 'react-native-inappbrowser-reborn';
+import {notifyBugsnag} from '../utils/bugsnag-utils';
+
+export async function openInAppBrowser(
+  url: string,
+  dismissButtonStyle: 'cancel' | 'close' | 'done',
+  options?: InAppBrowserOptions,
+) {
+  const previousStatusBarStyle = StatusBar.pushStackEntry({
+    barStyle: 'dark-content',
+    animated: true,
+  });
+  try {
+    await InAppBrowser.open(url, {
+      animated: true,
+      dismissButtonStyle,
+      // Android: Makes the InAppBrowser stay open after the app goes to
+      // background. Needs to be true for Vipps login and BankID.
+      showInRecents: true,
+      ...options,
+    });
+  } catch (error: any) {
+    notifyBugsnag(error);
+  }
+  StatusBar.popStackEntry(previousStatusBarStyle);
+}
+
+/**
+ * @returns true if the InAppBrowser was available, false otherwise
+ */
+export async function closeInAppBrowser() {
```

Comment:
Maybe not return anything? It doesn't seem we use the return value for anything now. And if we don't return anything we also don't need to have it async.

I personally find "fire-and-forget" functions a delight to use 😅 Just invoke it and no more troubles. When a function returns `Promise<boolean>` I'm like "Doh, here is some stuff that must be handled".

Reply:
Good point, let's to that 😄